### ISSUE TITLE
Remove hover effect on code block arrows in the gutter

### ISSUE THREAD

**Issue by flukeout** [mozilla/thimble.mozilla.org#1874]
Notice the strange hover behavior on the arrows in the gutter? The light up when you hover over the code but get fainter when you hover over the gutter, that seems to be the reverse of what we want.

![arrow-hover](https://cloud.githubusercontent.com/assets/25212/23867268/96a69032-07d8-11e7-8fcd-b48b680d0563.gif)

Let's...

* Remove the color change hover effect
* Make the cursor a pointer when you hover over arrows

**Comment by geekodour** [comment]
I tried fixing it, but that part of the editor comes from the  Bramble HTML editor, which comes under mozilla brackets  https://github.com/mozilla/brackets. So to fix this issue, I'll have to fix it either, we can directly add styles overwriting current styles else have to make changes to the upstream library.
Can anyone show some direction on this?

**Comment by humphd** [comment]
You're exactly right, it needs to happen in Brackets, not Thimble.  We just tend to file the bugs here, because that's what our users know.

Likely you need to fix the classes mentioned here: https://github.com/mozilla/brackets/blob/master/src/extensions/default/CodeFolding/main.js#L282-L297 and defined in https://github.com/mozilla/brackets/blob/master/src/extensions/default/CodeFolding/main.less

**Comment by sanariaz154** [comment]
I want to resolve this issue as I am preparing for GSOC17. can you please tell me where to start

**Comment by flukeout** [comment]
Hi @sanariaz154 - I think @geekodour is working on at at the moment. I'd recommend looking for another issue labelled "Good first bug" - thanks!

**Comment by sanariaz154** [comment]
@flukeout  can you please suggest me another good first bug or easy issue to which no contributor is assigned yet ?

**Comment by flukeout** [comment]
How about this one @sanariaz154 https://github.com/mozilla/thimble.mozilla.org/issues/1875

**Comment by geekodour** [comment]
I fixed it locally, I am not sure if it worked. I'll be sending the PR soon. I was getting build errors, It was strange. First of all, after doing **npm install** the package.json gets modified. (image) don't know if it's intended behavior, after that **npm run build** fails on **grunt build-browser**. 
`{ Error: Error: ENOENT: no such file or directory, open '/home/geekodour/OpenSource/brackets/src/
thirdparty/text/text.js'`
adding text.js pulls up another unknown error.
I tried fixing the issues without completing the build, I should work according to me. I'll be sending the PR in a while.
